src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/mod.rs

```diff
@@ -176,24 +180,8 @@ impl ExplorerProcess {
 
 impl Drop for ExplorerProcess {
     fn drop(&mut self) {
-        let output = if let Some(mut handler) = self.handler.take() {
-            let _ = handler.kill();
-            handler.wait_with_output().unwrap()
-        } else {
-            return;
-        };
-
-        if std::thread::panicking() {
-            if let Some(logs_dir) = &self.configuration.logs_dir {
-                println!(
-                    "persisting explorer logs after panic: {}",
-                    logs_dir.display()
-                );
```

Comment:
This was not displaying the path the logs would end up at. I believe the temporary file is moved to `/tmp/jormungandr_*/explorer.log` after it's written so it seemed ok for this to be removed (got confused by this log some times).

Reply:
What do you think about catching the stderr as well like we do here https://github.com/input-output-hk/catalyst-core/blob/cf46276b599c26594c6c5f563f5d02fb0d00c69c/src/jormungandr/testing/jormungandr-integration-tests/src/jormungandr/explorer/explorer_sanity.rs#L203 ?

Reply:
Sure. Nice catch